src/main/java/frc/robot/subsystems/loader/LoaderPID.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot.subsystems.loader;
+
+import java.util.function.DoubleSupplier;
+import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.controller.SimpleMotorFeedforward;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpiutil.math.MathUtil;
+import frc.robot.Robot;
+
+import static frc.robot.Robot.robotConstants;
+
+public class LoaderPID extends CommandBase {
+  private PIDController pidController;
+  private DoubleSupplier desiredVelocity;
+  private boolean isTuning;
+  private SimpleMotorFeedforward feedforward;
+
+  /**
+   * This does PID on the loader based on the desired velocity
```

Comment:
* This does PID on the loader based on the desired velocity
in which units vel? and does PID isn't a good explanation